client/src/components/UserRegisterForm.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Grid } from "semantic-ui-react";
+
+export class UserRegisterForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      name: "",
+      password: "",
+      confirmationPassword: "",
+      userSignUp: false
+    };
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = e => {
+    e.preventDefault();
+    const { password, confirmationPassword } = this.state;
+    if (password !== confirmationPassword) {
+      alert("passwords dont match");
+    } else {
+      const UserRequest = {
+        method: "POST",
+        body: JSON.stringify({
+          email: this.state.email,
+          name: this.state.name,
+          password: this.state.password,
+          confirmationPassword: this.state.confirmationPassword
+        }),
+        headers: {
+          "Content-Type": "application/json"
+        }
+      };
+      fetch("http://localhost:4000/auth/register", UserRequest)
+        .then(res => res.json())
+        .then(this.resetForm);
+    }
+  };
+  resetForm = () => {
+    this.setState({
+      name: "",
+      email: "",
+      password: "",
+      confirmationPassword: "",
+      userSignUp: "true"
+    });
+  };
+  render() {
+    let userSignUp;
+    if (this.state.userSignUp) {
+      userSignUp = "User Logged in successfully";
```

Comment:
After submitting the form, the user will not be logged in, just their account is created. So the message could be: "Your account has been successfully created."